bluepill/src/BPPacker.m

```diff
@@ -40,7 +40,7 @@ @implementation BPPacker
                      "Perhaps you forgot to 'build-for-testing'? (Cmd + Shift + U) in Xcode.");
         return NULL;
     }
-    NSMutableDictionary *testsToRunByTestFilePath = [[NSMutableDictionary alloc] init];
+    NSMutableDictionary *testsToRunByFilePath = [[NSMutableDictionary alloc] init];
```

Comment:
Unless the value is polymorphic (which I don't think it is here), let's add types for the values as well. I.e.

```
NSMutableDictionary <NSString *> *testsToRunByFilePath = ...
```